README.md

```diff
@@ -20,6 +20,20 @@ A convenience package that:
 
 Use this package to avoid having to explicitly install `leveldown` when you just want plain old `LevelDB` from `levelup`.
 
+* [Usage](#usage)
+* [API](#api)
+* [Promise Support](#promise-support)
+* [Events](#events)
+* [Contributing](#contributing)
+* [Contributors](#contributors)
+* [Backers](#backers)
+* [Sponsors](#sponsors)
+* [License](#license)
+
+**Note** that `level` only supports Node.js. There is ongoing work to make it work in both Node.js and in the browser. In the meantime, we suggest you to use [`level-browserify`](https://github.com/level/level-browserify).
```

Comment:
Maybe move this above the TOC, to "Use this package ... Note that level only".

Other than that, LGTM.